cli/interactive_tests/history

```diff
@@ -0,0 +1,69 @@
+#! /usr/bin/env expect -f
+
+# We are sending vt100 terminal sequences below, so inform readline
+# accordingly.
+set env(TERM) vt100
+
+# Keep the history in a test location, so as to not override the
+# developer's own history file.
+set env(COCKROACH_SQL_CLI_HISTORY) ".crdb_test_history" 
+
+# Everything in this test should be fast. Don't be tolerant for long
+# waits.
+set timeout 1
+
+system "rm -f ~/.crdb_test_history"
```

Comment:
Can we put this somewhere other than the user's home directory? This would be another good reason to run the test in docker; you wouldn't have to manually isolate the test environment.